unit-test/background/classes/score.es6.js

```diff
@@ -22,3 +77,42 @@ describe('score', () => {
         })
     })
 });
+
+describe('isaMajorNetwork', () => {
+    isaMajorNetworkTestCases.forEach((test) => {
+        it(`should return ${test.descr}`, () => {
+            spyOn(utils, 'findParent').and.returnValue(test.parent)
+            score = new Score(test.specialPage, test.domain)
+ 
+            if (test.parent) {
+                let slicedDomain = test.domain.split('.')
+                expect(utils.findParent).toHaveBeenCalledWith(slicedDomain)
+            }
+
+            let result = score.isaMajorTrackingNetwork
+
+            if (test.isZeroResult) {
+                expect(result).toEqual(0)
+            } else {
+                expect(result).toBeGreaterThan(0)
+            }
+        })
+    })
+});
+
+describe('getTosdr', () => {
+    tosdrTestCases.forEach((test) => {
+        it(`should return ${test.descr}`, () => {
+            score = new Score('', test.domain)
+ 
+            let result = score.tosdr
+            let message = result.message
+
+            if (test.isMessageBad) {
+                expect(message).toEqual('Bad')
+            } else {
+                expect(message).not.toEqual('Bad')
```

Comment:
Kept the test minimal here since tosdr is updated at every deploy, like the entityMap. Whether a site has bad practices seems the safest to test, especially with the chosen domains.